.github/workflows/run-testing.yaml

```diff
@@ -75,10 +79,11 @@ jobs:
       - name: Set up chart-testing
         uses: helm/chart-testing-action@v2.6.1
 
-      - name: Create k3d cluster
-        uses: nolar/setup-k3d-k3s@v1
+      - name: Create Kind cluster
+        uses: helm/kind-action@v1
         with:
-          version: ${{ matrix.k8s_version }}
+          node_image: kindest/node:${{ matrix.k8s_version }}
+          kubectl_version: ${{ matrix.k8s_version }}
```

Comment:
What is the reason to switch from one to another?

Reply:
nolar/setup-k3d-k3s@v1 action fails frequently